frontend/src/components/Filter.tsx

```diff
@@ -105,9 +105,14 @@ export class Filter extends React.Component<Filter.Props, {}> {
   };
 
   private onWindowKeyUp = (event: KeyboardEvent) => {
+    // Ignore if control key is being pressed
     if (event.ctrlKey) {
       return;
     }
+    // Ignore events dispatched to other elements that want to use it
+    if (['INPUT', 'TEXTAREA'].includes((event as any).target.tagName)) {
```

Comment:
Would have thought the `React.KeyboardEvent` would have the `target` field attached to it, or atleast inherited.

Reply:
Perhaps I just couldn't find the right type (I always forget what types I need for these things); the ones I tried all assumed that target could be null, and then had very few definite props for it. I'll have another look on Monday!

Reply:
I had a bit of a look and couldn't offhand find a type that lined up with the actual properties I expect here; it looks like the react event type simply doesn't commit to much about the target (probably because it can be any element and they didn't want to guarantee any specific properties would exist in it).

I did tweak the code to be more defensive incase `target` really doesn't exist (and tagName isn't a thing)

Reply:
lgtm, thanks for looking that up :) Yea I tend to forget all the types built into React often, so this was a nice little refresher.